facade-app/src/test/AboutUs.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+
+import AboutUs from "./../Components/AboutUs";
+
+describe("About Us", () => {
+  it("renders correctly", () => {
+    const { container } = render(<AboutUs />);
+
+    expect(container).toMatchSnapshot();
```

Comment:
One question, do we need to really match snapshot with a rendered element? can we compare a few logical components like you are doing it below?